cmd/monaco/integrationtest/v2/settings_integration_test.go

```diff
@@ -429,6 +429,48 @@ func TestOrdered_InsertAtFrontAndBackWorks(t *testing.T) {
 	})
 }
 
+func TestOrdered_InsertAtFrontAndBackWorksDeployTwice(t *testing.T) {
+	const configFolder = "test-resources/settings-ordered/insert-position"
+
+	const manifestFile = configFolder + "/manifest.yaml"
+
+	const specificEnvironment = "platform"
+	const project = "both-back-and-front-are-set-deploy-twice"
+	const schema = "builtin:url-based-sampling"
+
+	RunIntegrationWithCleanup(t, configFolder, manifestFile, specificEnvironment, "InsertAtBackWorks", func(fs afero.Fs, tc TestContext) {
+		pgiMeId := randomMeID("PROCESS_GROUP_INSTANCE")
+		setTestEnvVar(t, "MONACO_TARGET_ENTITY_SCOPE", pgiMeId, tc.suffix)
+		t.Log("Monitored entity ID for testing ('MONACO_TARGET_ENTITY_SCOPE') =", pgiMeId)
+
+		// first
+		err := monaco.Run(t, fs, fmt.Sprintf("monaco deploy %s --project %s --verbose", manifestFile, project))
+		require.NoError(t, err)
+		// second
+		err = monaco.Run(t, fs, fmt.Sprintf("monaco deploy %s --project %s --verbose", manifestFile, project))
+		require.NoError(t, err)
+
+		integrationtest.AssertAllConfigsAvailability(t, fs, manifestFile, []string{project}, specificEnvironment, true)
```

Comment:
Why do we need this call to `integrationtest.AssertAllConfigsAvailability(...)` (and if so, why only once)?

Reply:
Reflecting, I guess it might be something to check in our tests in general: if we are explicitly checking deployment manually we might not need this.

Reply:
Is there a reason you are concerned about having it in?